packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -128,8 +128,11 @@ const selectPatterns = createSelector(
 			patterns: themePatterns,
 			isResolving: isResolvingThemePatterns,
 		} = selectThemePatterns( select );
-		const { patterns: userPatterns, isResolving: isResolvingUserPatterns } =
-			selectUserPatterns( select );
+		const {
+			patterns: userPatterns,
+			isResolving: isResolvingUserPatterns,
+			categories: userPatternCategories,
```

Comment:
This is not needed or used.

Reply:
It's used in the `hasCategory` callback down here.

Reply:
This is quite hard to follow.. okay.